src/python/grpcio_tests/tests/unit/_alts_credentials_test.py

```diff
@@ -0,0 +1,56 @@
+# Copyright 2020 The gRPC Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Test of RPCs made using ALTS credentials."""
+
+import unittest
+import os
+from concurrent.futures import ThreadPoolExecutor
+import grpc
+
+
+class _GenericHandler(grpc.GenericRpcHandler):
+
+    def service(self, handler_call_details):
+        return grpc.unary_unary_rpc_method_handler(
+            lambda request, unused_context: request)
+
+
+class ALTSCredentialsTest(unittest.TestCase):
+
+    def _create_server(self):
+        server = grpc.server(ThreadPoolExecutor())
+        server.add_generic_rpc_handlers((_GenericHandler(),))
+        return server
+
+    @unittest.skipIf(os.name == 'nt',
+                     'TODO(https://github.com/grpc/grpc/issues/20078)')
```

Comment:
Does ALTS underlying uses `fd` like `LocalCredentials` (Windows only have socket handles)? Otherwise, it should not share the same issue with `LocalCredentials`.

Reply:
indeed, I think this should not apply - removed.